### DISCUSSION TITLE
how to install in mac ?

### DISCUSSION THREAD

**Discussion by mickeyperlstein** [kounoike/obs-virtualbg#55] [Q&A]
Hi I am using a macbook pro BigSur v11.6

how do i install your filter in my setup ?

**Comment by sgtpepper43** [comment]
I found the answer on [a related issue](https://github.com/kounoike/obs-virtualbg/issues/47#issuecomment-992335026). Go to Applications, then right click on OBS and click `Show Package Contents`. Then put `obs-virtualbg.so` (currently in the `bin` directory) into `Contents/Plugins`, and put everything in `data` into `Contents/Resources/data/obs-plugins/obs-virtualbg`

**Comment by kounoike** [comment]
> Currently, this plugin supports only Windows 64bit environment.

But, soon I will released next version which support on Mac

**Comment by aaronmergerequest** [comment]
Hey friends. Any update on this? It's December 2024 and there is [supposedly Mac support](https://github.com/kounoike/obs-virtualbg/pull/71) but no instructions on how to use.

**Comment by aaronmergerequest** [comment]
In the case that this doesn't end up working out for mac users, [obs-mac-background-removal](https://github.com/gxalpha/obs-mac-backgroundremoval/tree/0.2.0) "just works" and uses the built-in macOS Vision API. Install the package, restart OBS, find in effects.
